src/linker/pipeline.py

```diff
@@ -121,17 +121,24 @@ def write_implementation_rules(
         validation_file = str(
             results_dir / "input_validations" / implementation.validation_filename
         )
+        resources = (
```

Comment:
nit: we may want to specify this as "slurm_resources" (as opposed to spark resources or in the future who knows what else)

Unless your thought is the logic will go here eventually regardless of resource type?

Reply:
Yeah, I think the more general "resources" make sense at least on the rule side, given that we may support non-slurm execution resources, and this would be the place that those would go (though not necessarily for something like spark,)
https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#snakefiles-standard-resources